docs: document deferBelowFold setting disabled in 9.4.3 (#275892)
elastic/kibana: docs/release-notes/index.md

* Increases the maximum number of combined panels, sections, and controls on a dashboard from 100 to 1,000 [#272931]({{kib-pull}}272931).
* Fixes **Save and return** not working for Maps visualizations opened from the **Visualize** library [#274002]({{kib-pull}}274002).
* Fixes a stale closure in the Canvas autoplay timer that could skip pages after slides are added or removed [#268398]({{kib-pull}}268398).

> * Disables the experimental **Defer loading panels below "the fold"** dashboard setting (`labs:dashboard:deferBelowFold`) because enabling it could cause some dashboards to fail to load. The toggle remains in **Advanced Settings** but currently has no effect [#275632]({{kib-pull}}275632).

Author: ```suggestion
* Disables the experimental **Defer loading panels below "the fold"** dashboard setting (`labs:dashboard:deferBelowFold`) because enabling it can cause some dashboards to fail to load. The option remains in **Advanced Settings** but has no effect from this version [#275632]({{kib-pull}}275632).
```

> * Disables the experimental **Defer loading panels below "the fold"** dashboard setting (`labs:dashboard:deferBelowFold`) because enabling it can cause some dashboards to fail to load. The option remains in **Advanced Settings** but has no effect from this version [#275632]({{kib-pull}}275632).

Author: ```suggestion
* Disables the experimental **Defer loading panels below "the fold"** dashboard setting (`labs:dashboard:deferBelowFold`) because enabling it can cause some dashboards to fail to load. The option remains in **Advanced Settings** but has no effect from this version. It might be enabled again in a future version [#275632]({{kib-pull}}275632).
```
